### PR TITLE
Svg rendering

### PR DESCRIPTION
**Pull request by 1-Harshit**
for #28  also #9

### CODE REVIEW

File: src/components/SVGImage/index.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Image from "./SvgImage";
+
+export default Image;
```
Comment (bhuvansingla) [inline src/components/SVGImage/index.ts:3]: Add newline

File: src/components/SVGImage/image.source.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import COIN from "../../assets/images/svg/COIN.svg";
```
Comment (bhuvansingla) [inline src/components/SVGImage/image.source.ts:1]: `COIN` => `CoinTypographyLogo` everywhere

File: src/components/SVGImage/image.source.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+import Icon from "../../assets/images/svg/Icon.svg";
```
Comment (bhuvansingla) [inline src/components/SVGImage/image.source.ts:2]: `Icon` => `CoinLogo` everywhere

File: src/components/SVGImage/SvgImage.tsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  size?: string;
+  style?: StyleProp<ImageStyle>;
+}
+
+const SImage: React.FC<Prop> = (props) => {
+	const { name, size, style } = props;
+	// SVG Tag
+	let SVG = source.Crow;
+
+	const Size = size || "100%";
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:22]: `size = size || "100%";`

File: src/components/SVGImage/SvgImage.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+
+	
+	if (name === "crow") {
+		return <SVG height={Size} style={[styles.default, styles.crow, style]} />;
+	}
+
+	if (name === "COIN") {	
+		SVG = source.COIN;
+		return <SVG height={Size} style={[styles.default, styles.coin, style]} />;
+	} else {
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:32]: Write it as an `if` condition

File: src/components/SVGImage/SvgImage.tsx [hunk 6]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { StyleProp, ImageStyle } from "react-native";
+
+import source from "./image.source";
+import Style from "./image.styles";
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:5]: `Style` => `styles`

**Review by bhuvansingla — CHANGES_REQUESTED** [review]

File: src/components/SVGImage/SvgImage.tsx (hunk 5 above)
Comment (1-Harshit) [reply src/components/SVGImage/SvgImage.tsx:32]: Why?

File: src/assets/images/svg/COIN.svg [hunk 7]
```diff
@@ -0,0 +1,22 @@
+<svg width="402" height="148" viewBox="0 0 402 148" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (bhuvansingla) [inline src/assets/images/svg/COIN.svg:1]: Change file names here also.

File: src/components/SVGImage/SvgImage.tsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+	if (name === "CoinTypographyLogo") {	
+		SVG = source.CoinTypographyLogo;
+		return (
+      <SVG
+        height={height}
+        style={[styles.default, styles.coinTypographyLogo, style]}
+      />
+    );
+	} if(name === "CoinLogo") {
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:33]: Move to next line

File: src/components/SVGImage/SvgImage.tsx [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+		return (
+      <SVG
+        height={height}
+        style={[styles.default, styles.coinTypographyLogo, style]}
+      />
+    );
+	} if(name === "CoinLogo") {	
+		SVG = source.CoinLogo;
+		return <SVG height={height} style={[styles.default, styles.coinLogo, style]} />;
+	} else {
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:36]: else is unnecessary, just write return the return statement

File: src/components/SVGImage/SvgImage.tsx [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+	
+	if (name === "Crow") {
+		return <SVG height={height} style={[styles.default, styles.crow, style]} />;
+	}
+
+	if (name === "CoinTypographyLogo") {	
+		SVG = source.CoinTypographyLogo;
+		return (
+      <SVG
```
Comment (bhuvansingla) [inline src/components/SVGImage/SvgImage.tsx:28]: indentation

File: src/components/SVGImage/image.source.ts [hunk 11]
```diff
@@ -0,0 +1,9 @@
+import CoinTypographyLogo from "../../assets/images/svg/CoinTypographyLogo.svg";
+import CoinLogo from "../../assets/images/svg/CoinLogo.svg";
+import Crow from "../../assets/images/svg/Crow.svg";
+
+export default {
+	CoinTypographyLogo,
+	CoinLogo,
+	Crow,
+};
```
Comment (bhuvansingla) [inline src/components/SVGImage/image.source.ts:9]: Add new line

**Review by bhuvansingla — CHANGES_REQUESTED** [review]

**Review by bhuvansingla — APPROVED** [review]

**Pull request merged**